index.d.ts

```diff
@@ -8,6 +8,22 @@ declare namespace findUp {
 		readonly cwd?: string;
 	}
 
+	interface FullOptions extends Options {
```

Comment:
I don't know TypeScript that well, could this be imported from `locate-path` (and if it can should it)?

Reply:
You can import it at line 1:

```ts
import {Options as LocatePathOptions} from 'locate-path';
```

And then extend it on line 2:

```ts
interface Options extends LocatePathOptions {
```

Reply:
Actually `Options` is for using a `matcher` function where only `cwd` is accepted.

For the normal mode technically locatePath.AsyncOptions is supported, though I'm not sure we want TS to allow `concurrency` and `preserveOrder` to be used?  For now my patch is using locatePath.AsyncOptions but can easily be switched to locatePath.Options.

Also instead of extending the `locate-path` options I just directly used them since this module does not add any new options.